source/features/fix-no-pr-search.tsx

```diff
@@ -5,11 +5,14 @@ import features from '../feature-manager.js';
 import SearchQuery from '../github-helpers/search-query.js';
 
 function redirectToIssues(event: DelegateEvent<Event, HTMLFormElement>): void {
+	event.preventDefault();
 	const form = event.delegateTarget;
-	const searchQuery = new SearchQuery(form.elements['js-issues-search'].value);
+	const query = SearchQuery.from(location)
+	query.set(form.elements['js-issues-search'].value);
 
-	if (!searchQuery.includes('is:pr')) {
+	if (!query.includes('is:pr')) {
 		form.action = form.action.replace(/\/pulls$/, '/issues');
+		form.submit();
```

Comment:
You most likely want to move it here

```suggestion
		// Prevent submission via AJAX and use .submit() to allow the change from /pulls to /issues
		event.preventDefault();
		form.submit();
```

Reply:
Learn a lot from you! Appreciate it :P